Makefile

```diff
@@ -0,0 +1,39 @@
+ # Go parameters
+GOCMD=go
+GOBUILD=$(GOCMD) build
+GOCLEAN=$(GOCMD) clean
+GOTEST=$(GOCMD) test
+GOGET=$(GOCMD) get
+DOCKER=docker
+BINARY_NAME=adapter
+TARGET=target
+BINARY_LINUX=$(TARGET)/$(BINARY_NAME)_linux
+BINARY_DARWIN=$(TARGET)/$(BINARY_NAME)_darwin
+BINARY_WINDOWS=$(TARGET)/$(BINARY_NAME)_windows.EXE
+
+all: build test
+build: 
+	$(GOBUILD) -o $(BINARY_NAME) -v
+test: 
+	$(GOTEST) -v ./...
+clean: 
+	$(GOCLEAN)
+	rm -f $(BINARY_NAME)
+	rm -rf $(TARGET)
+run:
+	$(GOBUILD) -o $(BINARY_NAME) -v ./...
+	./$(BINARY_NAME)
+deps:
+	$(GOGET) github.com/wavefrontHQ/prometheus-storage-adapter    
```

Comment:
Also goget `github.com/Sirupsen/logrus` and `github.com/prometheus/prometheus` as dependencies?

Reply:
Shouldn't be needed. Go get should pull them down by analyzing the imports in our source files. I tried to build this on a fresh VM and it pulled down the libs correctly.

Reply:
Okay adding `build:deps` should fix it then. It was failing for me as `build` (default Makefile target) doesn't do Goget currently.

Reply:
OK. The readme should have a reference to "deps". Will fix that.

Reply:
...actually, it already has. The readme file says ```make deps build```

Reply:
Fixed